experimental/Permutations/Permutations.jl

```diff
@@ -0,0 +1,136 @@
+# This file implements a new way to input permutations in Julia. For example
+# it is possible to create a permutation as follow
+# pi = Oscar.Permutations.@perm (1,2,3)(4,5)(6,7,8)
+# > (1,2,3)(4,5)(6,7,8)
+# For this we use macros to modify the syntax tree of (1,2,3)(4,5)(6,7,8) such that
+# Julia can deal with the expression.
+
+module Permutations
+
+ using GAP
+ using Oscar
+
+################################################################################
+# Macro to input a permutation as
+# julia> pi = Oscar.Permutations.@perm (1,2,3)(4,5)(6,7,8)
+# > (1,2,3)(4,5)(6,7,8)
+#
+macro perm(ex)
+    h = (ex).head
+    arg = (ex).args
+    res = []
+    
+    while h != :tuple
```

Comment:
Needs some more error testing. I would actually recommend to do
```suggestion
    while h == :call
```
and then after the loop verify that the final expression really is a `:tuple`